### PR TITLE
Added recipe for cup of sugar

### LINKED ISSUE #32: Need Recipe for cup of sugar
My daughter wanted it

### PR DESCRIPTION
**Pull request by paul-gallenkemper**
Fixes #32

### CODE REVIEW

**Comment by paul-gallenkemper** [comment]
Please accept this PR, it does not contain a trojan ;)

File: recipes/cup_of_sugar.qmd [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+amount_of_sugar <- sample(dice, 1)
+```
+
+## {{< fa kitchen-set >}} Procedure
+
+You need to put {{amount_of_sugar}} grams in a cup.
+
+## Photo of result
+
+![Zucker](https://www.happyballoon.de/thumbnail/4d/c1/d0/1701938245/63780351a985e5f5715ba9e630bf5d69ea268092_1920x1920.jpeg)\]
```
Comment (lnnrtwttkhn) [inline recipes/cup_of_sugar.qmd:24]: remove this: `\]`

**Comment by lnnrtwttkhn** [comment]
Thanks @paul-gallenkemper! I reviewed your contribution. Please fix the remaining issues listed in my review.

**Comment by paul-gallenkemper** [comment]
I resolved the issues and pushed again in my Git to my fork. Not sure whether these changes also appear in this pull request. 

See you next week.

**Comment by lnnrtwttkhn** [comment]
Hi @paul-gallenkemper, not quite. You made those additional changes on the `main` branch but the PR is from the `paul` branch in your forked repository. You can either (1) merge the `main` branch into the `paul` branch in your forked repo or (2) switch the PR from `paul` to `main` branch that incorporates both commits.

In addition, I noticed that there is still an error with the inline R code. I should be

```R
`r amount_of_sugar`
```

not


```R
'r amount_of_sugar'
```

Let me know how you want to proceed here. thanks!

**Comment by paul-gallenkemper** [comment]
Hey Lennart, ich habs nochmal auf dem Paul branch gefixt. Sieht ja so aus, als würde das dann automatisch in den PR einbauen

git ist echt genial, als hätte sich da jemand Gedanken gemacht...

**Comment by lnnrtwttkhn** [comment]
@allcontributors please add @paul-gallenkemper for bug, content, code

**Pull request merged**